pydrumscore/export.py

```diff
@@ -516,7 +517,7 @@ def add_note(chord, notedef):
                 if tuplet_counter == 0:
                     add_xml_elem("endTuplet", voice)
 
-    # Save
+    ######################### Save
```

Comment:
Comments that start with many '#' probably won't pass the linting step. You can just put a single '#' with a space.